jenkins-slaves/jenkins-slave-conftest/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM quay.io/openshift/origin-jenkins-agent-base:4.4
+
+ENV BATS_VERSION=1.2.0 \
+    YQ_VERSION=3.3.2 \
+    CONFTEST_VERSION=0.18.2
+
+RUN curl --fail -skL https://github.com/open-policy-agent/conftest/releases/download/v${CONFTEST_VERSION}/conftest_${CONFTEST_VERSION}_Linux_x86_64.tar.gz -o /tmp/conftest.tar.gz && \ 
+    tar -C /tmp -xzf /tmp/conftest.tar.gz && \
+    mv /tmp/conftest /usr/local/bin && \
+    curl --fail -sL  https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64 -o /usr/local/bin/yq && \
```

Comment:
FYI: the yq used by the GH-Action isn't this ya, its: https://pypi.org/project/yq/ - which also requires JQ

Reply:
JQ is pre installed on the slave from memory, but cheers for the heads up @garethahealy. I guess if your policies are using this python `yq` library i should update this pr to match